test/tests/problems/eigen_problem/initial_condition/tests

```diff
@@ -11,15 +11,6 @@
     requirement = "The system shall provide an initial guess to Newton if users request."
   [../]
 
-  [./output_inverse_eigenvalue]
-    type = 'CSVDiff'
-    input = 'ne_ic.i'
-    csvdiff = 'ne_ic_out_inverse_eigenvalues_0001.csv'
-    cli_args = 'Executioner/output_inverse_eigenvalue=true Outputs/file_base=ne_ic_out_inverse'
-    slepc_version = '>=3.8.0'
-    requirement = "The system shall provide an option to output the eigenvalue as its inverse."
-  [../]
```

Comment:
It would be nice if we still can test this feature.

Reply:
Any thought?

Reply:
We can add a command-line argument for MOOSE test app to turn on output inverse. I can add a quick commit mimic what we are going to do in Griffin.